x-pack/plugins/cases/public/components/case_view/components/case_view_alerts.tsx

```diff
@@ -61,7 +69,14 @@ export const CaseViewAlerts = ({ caseData }: CaseViewAlertsProps) => {
       </EuiFlexItem>
     </EuiFlexGroup>
   ) : (
-    triggersActionsUi.getAlertsStateTable(alertStateProps)
+    <EuiFlexItem data-test-subj="case-view-alerts">
+      <CaseViewTabs caseData={caseData} activeTab={CASE_VIEW_PAGE_TABS.ALERTS} />
+      <EuiFlexGroup>
+        <EuiFlexItem style={{ minHeight: 300 }}>
```

Comment:
let's use the `css` prop instead of `style`

Reply:
The `minHeight` adds a gray area below the table. Do we need the height?

<img width="670" alt="Screenshot 2023-02-22 at 5 01 38 PM" src="https://user-images.githubusercontent.com/7871006/220661550-ef63437a-2b96-4ffd-86d1-c45796ab5054.png">

Reply:
Without providing height, it crops last entry when multiple alerts are added:

<img width="1445" alt="image" src="https://user-images.githubusercontent.com/117571355/220663770-a884f46b-2d01-4ee4-8667-a90bfb79c297.png">

Reply:
I added min-height as it similar to what is suggested by [Eui](https://elastic.github.io/eui/#/tabular-content/data-grid) :

_When placed within an [EuiFlexGroup and EuiFlexItem](https://elastic.github.io/eui/#/layout/flex), the data grid will have trouble shrinking to fit. To fix this, you will need to manually add a style of min-width: 0 to the EuiFlexItem._